aws_lambda_powertools/utilities/parser/models/s3.py

```diff
@@ -11,88 +11,319 @@
 
 class S3EventRecordGlacierRestoreEventData(BaseModel):
     lifecycleRestorationExpiryTime: datetime
-    lifecycleRestoreStorageClass: str
+    lifecycleRestoreStorageClass: str = Field(
+        description="Source storage class for restore.",
+        examples=[
+            "standard",
+            "standard_ia",
+            "glacier",
+        ],
+    )
 
 
 class S3EventRecordGlacierEventData(BaseModel):
     restoreEventData: S3EventRecordGlacierRestoreEventData
 
 
 class S3Identity(BaseModel):
-    principalId: str
+    principalId: str = Field(
+        description="Amazon customer ID of the user who caused the event.",
+        examples=[
+            "AIDAJDPLRKLG7UEXAMPLE",
+            "A1YQ72UWCM96UF",
+            "AWS:AIDAINPONIXQXHT3IKHL2",
+        ],
+    )
 
 
 class S3RequestParameters(BaseModel):
     sourceIPAddress: Union[IPvAnyNetwork, Literal["s3.amazonaws.com"]]
```

Comment:
Can you add a description and an example here? Please let me know if you need help writing them.